### PR TITLE
Moving green house gas section to carbon section

### PR DESCRIPTION
**Pull request by jawache**
Moved the section on understanding the GHG protocol to the Carbon section. It's fundamental to everything and moving it earlier means we can mention it in the embodied carbon section.

### CODE REVIEW

File: Principles/Carbon.md [hunk 1]
```diff
@@ -70,6 +74,48 @@ We also need greenhouses gases in our everyday lives. For example CO<sub>2</sub>
 
 The earth has had many different compositions of greenhouse gases in its history, and natural fluctuation does occur over time (millennia). This is often refereed to as climate change, whereas global warming is separated out as the human influenced factor which has taken place over a very short period of time (decades).
 
+### How to measure Greenhouse Gas emissions?
```
Comment (greenhsu123) [inline Principles/Carbon.md:77]: Could we extend this paragraph to include a short overview of how the below 3 tie in with measuring GHG and GHG Protocol?

File: Principles/Carbon.md [hunk 2]
```diff
@@ -70,6 +74,48 @@ We also need greenhouses gases in our everyday lives. For example CO<sub>2</sub>
...
+
+The [Greenhouse Gas Protocol](https://ghgprotocol.org) is the most widely used and internationally recognized greenhouse gas accounting standard. In fact, [92%](https://ghgprotocol.org/about-us) of all Fortune 500 companies use the GHG protocol when calculating and disclosing their carbon emissions.
```
[the remark this answers is not in the document]
Comment (jawache) [reply Principles/Carbon.md:79]: It's best to go with US, British spelling will never win 🤷🏽‍♂️

File: Principles/Carbon.md (hunk 1 above)
Comment (jawache) [reply Principles/Carbon.md:77]: Sounds good but I've got super limited time right now. Do you want to take a stab at this @greenhsu123?

File: Principles/Carbon.md [hunk 3]
```diff
@@ -70,6 +74,48 @@ We also need greenhouses gases in our everyday lives. For example CO<sub>2</sub>
 
 The earth has had many different compositions of greenhouse gases in its history, and natural fluctuation does occur over time (millennia). This is often refereed to as climate change, whereas global warming is separated out as the human influenced factor which has taken place over a very short period of time (decades).
 
+### How to measure Greenhouse Gas emissions?
+
+The [Greenhouse Gas Protocol](https://ghgprotocol.org) is the most widely used and internationally recognized greenhouse gas accounting standard. In fact, [92%](https://ghgprotocol.org/about-us) of all Fortune 500 companies use the GHG protocol when calculating and disclosing their carbon emissions. 
+
+The GHG Protocol divides emissions into 3 scopes:
+
+* Scope 1: Direct emissions related to on-site fuel combustion or fleet vehicles;
```
[the remark this answers is not in the document]
Comment (jawache) [reply Principles/Carbon.md:83]:
```suggestion
* Scope 1: Direct emissions from operations owned or controlled by the reporting organisation, such as on-site fuel combustion or fleet vehicles.
```

**Pull request merged**